src/component/BarreDeRecherche.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+const axios = require('axios');
+var parseString = require('xml2js').parseString;
+
+class BarreDeRecherche extends React.Component {
+    RechercheImage(event){
+    
+        event.preventDefault();
```

Comment:
@malvena15 What would happen if we didn't prevent the default behavior?

Reply:
My page gonna refresh and the information a get from my input will not be kept. So nothing will append